rosbag2_compression/test/rosbag2_compression/test_sequential_compression_writer.cpp

```diff
@@ -241,14 +241,14 @@ TEST_F(SequentialCompressionWriterTest, writer_creates_correct_metadata_relative
   message->topic_name = test_topic_name;
 
   writer_->write(message);
-  // bag size == max_bafile_size, no split yet
+  // bag size == max_bafile_size, split
```

Comment:
@MichaelOrlov nit^2: I know it's preexisting, but `max_bafile_size` -> `max_bagfile_size`

Reply:
Thanks for catching it up. Fixed via amend commit.